modules/ossm-rn-fixed-issues.adoc

```diff
@@ -66,6 +66,10 @@ $ oc delete crd challenges.certmanager.k8s.io
 
 * link:https://issues.jboss.org/browse/KIALI-3239[KIALI-3239] If a Kiali Operator pod has failed with a status of “Evicted” it blocks the Kiali operator from deploying. The workaround is to delete the Evicted pod and redeploy the Kiali operator.
 
+* link:https://issues.jboss.org/browse/KIALI-3118[KIALI-3118] After changes to the ServiceMeshMemberRoll, for example adding or removing projects, the Kiali pod restarts and then displays errors on the Graph page while the Kiali pod is restarting.
```

Comment:
s/pod/Pod/
We aren't 100% consistent in OCP docs, but this is in our guidance: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting
But if it's going to look incongruous against the rest of your doc set, then maybe leave it as is.

Reply:
I'm going to leave that cleanup for another day, we've got pod(s) lowercase all through our docs.